test/suite/test_tiered18.py

```diff
@@ -69,16 +69,28 @@ def check_metadata(self, uri, val_str):
     def test_tiered_shared(self):
         self.pr("create tiered shared with default")
         base_create = 'key_format=S,value_format=S'
-        self.session.create(self.uri_non_shared, base_create)
-        self.check_metadata(self.uri_non_shared, 'key_format=S')
+        self.session.create(self.uri_shared_default, base_create)
+        self.check_metadata(self.uri_shared_default, 'key_format=S')
         self.check_metadata("colgroup:test_tiered18_shared_default.active", 'file:test_tiered18_shared_default.wt')
         self.check_metadata("colgroup:test_tiered18_shared_default.shared", 'tiered:test_tiered18_shared_default')
-        self.session.drop(self.uri_non_shared)
+        self.check_metadata("file:test_tiered18_shared_default.wt", 'log=(enabled=true)')
+        self.check_metadata("tiered:test_tiered18_shared_default", 'log=(enabled=true)')
```

Comment:
This would be better to be defined up top using something like `shared_base=test_tiered18_shared_default` and then have `uri_shared_default` and `file_shared_default` and `tiered_shared_default` etc so that if the name ever changes it is one location to modify. Same with the others.

Reply:
I updated the test as suggested. Please check.